src/lib.rs

```diff
@@ -1,5 +1,5 @@
 //! The `wolfssl` crate is designed to be a Rust layer built on top of
-//! the `wolfssl-sys` crate (a C passthrough crate).
+//! the `wolfssl-s   ys` crate (a C passthrough crate).
```

Comment:
Typo here? Extra space inserted in crate name

Reply:
Hey thanks Brendan, 
I made that change while in draft mode, to trigger a new build and check caching was working properly and the changes were properly compiled.
Rolled it back before marking the PR ready for review.